setup.py

```diff
@@ -49,7 +49,7 @@
     packages=find_packages(),
     install_requires=[
         "jupyter_core>=4.0,<5.0",
-        "kfp==0.5.1",
+        "kfp==1.0.0",
         "urllib3==1.24.2",
```

Comment:
@akchinSTC FYI, what are the implications here? I thought we pinned to 0.5.1 for some reason that I don't remember...

Reply:
@akchinSTC said it's ok